bigquery/simpleapp/simpleapp.go

```diff
@@ -53,16 +54,16 @@ func query(proj string) (*bigquery.RowIterator, error) {
 				CAST(id as STRING)) as url,
 			view_count
 		FROM ` + "`bigquery-public-data.stackoverflow.posts_questions` " +
-		`WHERE tags like '%google-bigquery%'
+			`WHERE tags like '%google-bigquery%'
```

Comment:
maybe move the backtick up to the previous line so you can line it all up?

```
FROM ` + "`...` " + `
WHERE ...
```

Reply:
Yeah, done.